test/project_types/extension/commands/extension_command_test.rb

```diff
@@ -40,21 +36,16 @@ def test_extension_type_aborts_if_a_lazily_initialized_field_of_an_unknown_type_
         ])
       end
 
-      def test_extension_type_returns_a_lazy_specification_handler
-        setup_temp_project
```

Comment:
Were you able to verify that removing `setup_temp_project` in this test and other tests below does not break the tests. Arguably, if we can take this code out, that would be great because it is clearly at best executed for its side effects and at worst completely unnecessary. 

In short, just double check that this didn't do anything important and it is indeed safe to remove. 😄